docs/design/autoware-interfaces/ad-api/features/interface.md

```diff
@@ -0,0 +1,17 @@
+# Interface feature
+
+## Description
+
+Considering the product life cycle, there may be multiple vehicles using different versions of the AD API due to changes in requirements or functional improvements.
+For example, one vehicle uses `v1` for stability and another vehicle uses `v2` to enable more advanced functionality.
+
+In that situation, the AD API users such as developers of a web service have to switch the application behavior based on the version that each vehicle uses.
+The version of AD API follows [Semantic Versioning][semver] in order to provide an intuitive understanding of the changes between versions.
```

Comment:
Since this section is the design concept, I guess API users aren't interested in it.
So, I'll recommend keeping it in the msgs package as it is.

Reply:
Moved to message package readme.
https://github.com/autowarefoundation/autoware.universe/pull/1090